recipe/meta.yaml

```
@@ -12,15 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: python -m pip install . --no-deps --ignore-installed -vvv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vvv
```

remove --ignore-installed as suggested in https://github.com/conda-forge/psyplot-feedstock/pull/26#issuecomment-2034657632